hack/push-bundles.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/env bash
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Pushes policy bundles to quay.io, but only if anything changed since
+# the last bundle was pushed.
+#
+set -e -o pipefail
+
+REPO_ORG=hacbs-contract
+QUAY_API_URL=https://quay.io/api/v1/repository/$REPO_ORG
+ROOT_DIR=$( git rev-parse --show-toplevel )
+BUNDLES="release pipeline data"
+
+# The data bundle is a little different to the other two.
+# Encapsulate the differences these little functions.
+#
+function bundle_src_dirs() {
+  [ $1 == "data" ] && echo "data" || echo "policy/lib policy/$1"
+}
+
+function bundle_subdir() {
+  [ $1 == "data" ] && echo "data" || echo "policy"
+}
+
+function exclusions() {
+  [ $1 == "data" ] && echo "config.json" || echo "artifacthub-pkg.yml"
+}
+
+function repo_name() {
+  [ $1 == "data" ] && echo "ec-policy-data" || echo "ec-$1-policy"
+}
+
+function conftest_push() {
+  if [ $1 == "data" ]; then
+    conftest push --policy '' --data data $2 || true
```

Comment:
Why ignore errors?

Reply:
Good catch. I was mainly for my hacking since I was always getting the same error. I'll remove.